docs/uml-class-diagrams/display02/test/DisplayCommInterfaceImplementationTests/DisplayCommInterfaceImplementationTests.puml

```diff
@@ -2,8 +2,11 @@
 
 package edu.oakland.test.display02 {
     class DisplayCommInterfaceImplementationTests {
+        + void isRfidRecieved()
+        + void passToComm2WayisNotNull()
```

Comment:
Just for clarity, let's rename this to `displayComm2WayIsNotNull`, that way we know this doesn't actually need to test the passing.
```suggestion
        + void displayComm2WayIsNotNull()
```